actions/debootstrap_action.go

```diff
@@ -157,6 +158,24 @@ func (d *DebootstrapAction) RunSecondStage(context debos.DebosContext) error {
 	return err
 }
 
+// Guess if suite is something before usr-is-merged was introduced
+func (d *DebootstrapAction) isLikelyOldSuite() bool {
+	switch strings.ToLower(d.Suite) {
+	case "sid", "unstable":
+		return false
+	case "testing":
+		return false
+	case "bookworm":
```

Comment:
Note: In 2023 bookworm will be stable, and with the code as is using suite name "stable" will then start to fail.

Reply:
Maybe we should leave the stable case out of this PR?

Reply:
Yes, it is left out. What I'm saying is that "stable" needs to be added once bookworm becomes stable (and all users needs to upgrade their debos binary again).

Reply:
Hopefully everyone uses release names rather than stable/testing/unstable though....

Reply:
That's something we could add a warning for (i'd accept that as a PR on top) :-)